backend/spec/features/admin/quick_switch_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe "Quick Switch", type: :feature, js: true do
+  context 'as admin user' do
+    stub_authorization!
+
+    context "visiting any page" do
+      let(:initial_page) { spree.admin_orders_path }
+      before(:each) { visit initial_page }
+
+      it 'can open the quick switch' do
+        open_quick_switch
+        expect(page).to have_css("#quick-switch", visible: true)
```

Comment:
@tvdeyen after your review I just changed this line from:

```ruby
expect(page).to have_content("How do I use this") # TODO: check visible class
```

since I did left that `TODO` but totally forgot to fix it before pushing.

Reply:
👍